package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "esn-frontend-calendar-public",
+  "version": "0.1.0",
+  "description": "Calendar Web Frontend public page for the OpenPaaS Suite - https://open-paas.org/",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "serve": "webpack-dev-server --watch --progress",
+    "serve:prod": "webpack-dev-server --env=prod",
+    "build:dev": "webpack",
+    "build:prod": "webpack --env=prod"
+  },
+  "author": "",
+  "license": "AGPLv3",
+  "dependencies": {
+    "esn-frontend-common-libs": "github:openpaas-suite/esn-frontend-common-libs#main",
+    "esn-frontend-calendar": "github:openpaas-suite/esn-frontend-calendar#main",
+    "fullcalendar": "3.9.0"
```

Comment:
@tuanlc isn't it the same issue as yours: Required `fullcalendar` here when already required in esn-frontend-calendar module is bad

Reply:
Not really, mine is fullcalendar should be required in intermediary module `esn-frontend-inbox-calendar` instead of in inbox SPA.